docs/en/reference/transactions-and-concurrency.rst

```diff
@@ -106,7 +115,7 @@ functionally equivalent to the previously shown code looks as follows:
 
 .. warning::
 
-    For historical reasons, ``EntityManager#transactional($func)`` will return
+    For historical reasons, ``EntityManager#wrapInTransaction($func)`` will return
```

Comment:
Didnt the return value change in the new method? Afair it was introduced to fix these historical reasons.

Reply:
on v2.9 https://github.com/doctrine/orm/blob/2.19.5/src/EntityManager.php#L255
on v3.1 https://github.com/doctrine/orm/blob/3.1.x/src/EntityManager.php#L187

should i remove that `historical reason` ?